src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -834,18 +834,59 @@ int wm_vuldet_create_file(const char *path, const char *source) {
     const char *ROOT = "root";
     const char *sql;
     const char *tail;
-    sqlite3 *db = NULL;
-    sqlite3_stmt *stmt = NULL;
     int result;
     uid_t uid;
     gid_t gid;
+    bool already_removed = false;
+    bool db_existed = (access(path, F_OK) == OS_SUCCESS);
+
+opendb: ;
```

Comment:
```suggestion
opendb:
```

Reply:
Yes, this seems a mistake, but it's necessary  because the next initialization makes a compiler error.
https://en.cppreference.com/w/c/language/goto